### PR TITLE
added tests and updated defaults.

### PR DESCRIPTION
**Pull request by marshHawk4**

### CODE REVIEW

File: cdr_schemas/area_extraction.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ class Area_Extraction(BaseModel):
                     origin (0,0)."""
     )
     bbox: Optional[List[Union[float, int]]] = Field(
+        default=None,
```
Comment (scotthaleen) [inline cdr_schemas/area_extraction.py]: Remove optional and add a `default_factory=list`

File: cdr_schemas/area_extraction.py [hunk 2]
```diff
@@ -41,7 +42,7 @@ class Area_Extraction(BaseModel):
         """,
     )
     text: Optional[str] = Field(
-        ...,
+        default=None,
```
Comment (scotthaleen) [inline cdr_schemas/area_extraction.py]: required with default of `""`

File: cdr_schemas/feature_results.py [hunk 3]
```diff
@@ -20,25 +20,25 @@ class FeatureResults(BaseModel):
     )
     cog_id: str = Field(description="Cog id.")
     line_feature_results: Optional[List[LineLegendAndFeaturesResult]] = Field(
-        default=None,
+        default=[],
```
Comment (scotthaleen) [inline cdr_schemas/feature_results.py]: change to required and `default_factory=list`

File: cdr_schemas/area_extraction.py (hunk 2 above)
Comment (marshHawk4) [reply cdr_schemas/area_extraction.py]: Why can't this be None?

**Pull request merged**